modules/indexer/issues/elastic_search.go

```diff
@@ -0,0 +1,195 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package issues
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"time"
+
+	"code.gitea.io/gitea/modules/log"
+
+	"gopkg.in/olivere/elastic.v5"
+)
+
+var (
+	_ Indexer = &ElesticSearchIndexer{}
+)
+
+// ElesticSearchIndexer implements Indexer interface
+type ElesticSearchIndexer struct {
+	client      *elastic.Client
+	indexerName string
+	typeName    string
+}
+
+type elasticLogger struct {
+	*log.Logger
+}
+
+func (l elasticLogger) Printf(format string, args ...interface{}) {
+	l.Logger.Log(2, l.Logger.GetLevel(), format, args...)
```

Comment:
Are you sure this should be 2? That is you want to log as if you're logging from the thing, that called the thing, that called Printf?

Reply:
It's correct!